packages/yup/README.md

```diff
@@ -0,0 +1,98 @@
+# yup
+> Method extensions for the [yup](https://github.com/jquense/yup)
+
+[![Version](https://img.shields.io/npm/v/@availity/yup.svg?style=for-the-badge)](https://www.npmjs.com/package/@availity/yup)
+
+## Install
+
+```bash
+npm install @availity/yup yup --save
+```
+
+## Usage
+
+Import the package in the root of your project somewhere and you will have access to all of the provided yup methods.
+
+```javascript
+import '@availity/yup'
+import * as yup from 'yup';
+
+const schema = yup.string().dateFormat('MM/DD/YYYY');
+
+schema.isValid('12-12-2012');
+```
+
+## Table of Contents
+
+- [isRequired](#isRequired)
+- [dateRange](#dateRange)
+- [dateFormat](#dateFormat)
+- [between](#between)
+
+> Note any date validation methods will required [moment](https://www.npmjs.com/package/moment) to be installed
+
+
+
+## Methods
+### isRequired [**String**,**Array**,**Number**]
+#### Parameters
+- **required** - `boolean`. **required**. Whether or not the given string is required.
```

Comment:
says this is a required parameter, but the examples below do not always provide it.

Reply:
Good catch, forgot to update this after making changes to the ts defs and source.